index.html

```diff
@@ -1018,12 +1017,34 @@ <h4 class="metrics-label">Dana Scully</h4>
         </div>
     </div>
 
+    <h2>Dialogs/Modals</h2>
+
+    <a href="" id="sampleDialogLink">Launch</a>
+
+    <div id="sampleDialog" style="display:none">
+        <form>
+            <div class="block-content">
+                BLOCK CONTENT
+            </div>
+            <div class="block-footer">
+
+                <div class="submit-btn-group">
+                    <!-- cancels were formerly links so they can remain a-href's, or turn into buttons since they're now styled as buttons -->
+                    <a href="#" onclick="$('#sampleDialog').dialog('close'); return false;" class="btn">Cancel</a>
+                    <button class="btn primary-btn" type="button" onclick="$('#sampleDialog').dialog('close'); return false;">Done</button>
+                </div>
+
+            </div>
+        </form>
+    </div>
+
     </body>
 </html>
 
-<script   src="https://code.jquery.com/jquery-1.12.3.min.js"   integrity="sha256-aaODHAgvwQW1bFOGXMeX+pC4PZIPsvn2h1sArYOhgXQ="   crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
```

Comment:
It's important that we use the exact versions of jquery and jquery ui that we use in our app.